acceptance/lib/puppet/acceptance/install_utils.rb

```diff
@@ -232,6 +232,15 @@ def configure_gem_mirror(hosts)
           on host, "#{gem} source --add #{gem_source}"
         end
       end
+
+      def install_puppet_from_msi( host, opts )
+        if not link_exists?(opts[:url])
+          raise "Puppet does not exist at #{opts[:url]}!"
+        end
+
+        # beaker has already added puppetbindir to PATH in ~/.ssh/environment
+        on host, "cmd.exe /c start /w msiexec /qn /i #{opts[:url]}"
```

Comment:
After installing the MSI, we should execute `puppet --version` and `ruby --version` to ensure it was installed correctly, and so that later steps don't fail in obscure ways.